Gaming The Syllabus/Assets/Scripts/TooltipManager.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using TMPro;
+
+public class TooltipManager : MonoBehaviour
+{
+    public static TooltipManager _instance;
+
+    public TextMeshProUGUI textComponent;
+
+    void Awake()
+    {
+        if (_instance != null && _instance != this)
+        {
+            Destroy(this.gameObject);
+        }
+        else
+        {
+            _instance = this;
+        }
+    }
```

Comment:
Btw, the Singleton base class I use handles all of this checking so you don't have to write it everywhere in case you're wondering. Just inherit and call InitializeSingleton and all this should get done for you